W2Vec_occupations.py

```diff
@@ -0,0 +1,17 @@
+import gensim.models.keyedvectors as word2vec
+import pandas as pd
+
+# Load Google's pre-trained Word2Vec model.
+model = word2vec.KeyedVectors.load_word2vec_format('GoogleNews-vectors-negative300-SLIM.bin', binary=True)
+
+df_occupation = pd.read_csv('u.occupation', sep='\t', names=['occupation'], encoding='latin-1')
+df_occupation = df_occupation.values
+
+
+vec=[]
+for i in range(21):
```

Comment:
As you start to settle on your final product, the final step is to go through your code and remove these hard-coded parameters.

This would be something like:

for i in df_occupation:
   vec.append(model.similarity(i[0], 'scientist'))